.hintrc

```diff
@@ -34,4 +34,4 @@
             ]
         }
     ]
-}
+}
```

Comment:
Missing new line here. Can you please add it so there aren't any changes in this file?